doc/devel/development_workflow.rst

```diff
@@ -6,41 +6,34 @@
 Development workflow
 ####################
 
-You already have your own forked copy of the `Matplotlib`_ repository, by
-following :ref:`forking`. You have :ref:`set-up-fork`. You have configured
-git by following :ref:`configure-git`.  Now you are ready for some real work.
-
 Workflow summary
 ================
 
 In what follows we'll refer to the upstream Matplotlib ``main`` branch, as
 "trunk".
 
 * Don't use your ``main`` branch for anything.  Consider deleting it.
-* When you are starting a new set of changes, fetch any changes from trunk,
+* When you are starting a new set of changes, fetch any changes from ``main``,
   and start a new *feature branch* from that.
-* Make a new branch for each separable set of changes |emdash| "one task, one
-  branch" (`ipython git workflow`_).
+* Make a new branch for each separable set of changes — "one task, one
+  branch".
 * Name your branch for the purpose of the changes - e.g.
   ``bugfix-for-issue-14`` or ``refactor-database-code``.
-* If you can possibly avoid it, avoid merging trunk or any other branches into
-  your feature branch while you are working.
-* If you do find yourself merging from trunk, consider :ref:`rebase-on-trunk`
-* Ask on the `Matplotlib mailing list`_ if you get stuck.
-* Ask for code review!
+* If you get stuck, reach out on Gitter or
+  `discourse <https://discourse.matplotlib.org>`__.
+* Ask for a code review!
```

Comment:
```suggestion
* When your ready or need feedback on your code, open a pull-request
  so that the matplotlib developers can give feedback and eventually
  include your suggested code into the ``main`` branch.
```